src/app/actions.ts

```diff
@@ -11,7 +13,6 @@ export enum AppState {
 
 export enum Actions {
   SET_APP_STATE = 'APP/SET_APP_STATE',
-  SET_LOGGED_IN = 'APP/SET_LOGGED_IN',
```

Comment:
unrelated change but i felt bad about adding to this already big redux so felt good to be able to remove this, removing it here saves me a separate redux migration